seed_scheduler/settings.py

```diff
@@ -188,6 +188,9 @@
     'seed_scheduler.scheduler.tasks.queue_tasks': {
         'queue': 'priority',
     },
+    'seed_scheduler.scheduler.tasks.requeue_failed_tasks': {
+        'queue': 'priority',
+    },
```

Comment:
The reason I chose a different queue is because priority is used rarely and the workers sit idle most the day. We could create a new queue though and have the current priority workers handle that as well. What would you prefer?